app/views/discovery/_carousel_vendor_element.html.haml

```diff
@@ -1,7 +0,0 @@
-%div{:class => "container carousel_element #{local_assigns[:ownership].gsub(" ", "_")}", :id => local_assigns[:vendor].name.gsub(" ", "_"), :onclick => "routeToPageDirectly('#{vendor_path(local_assigns[:vendor].id)}')"}
```

Comment:
This has been replaced with the generic carousel_element correct?

Reply:
Yes, that's correct. David found that _carousel_vendor_element and _carousel_product_element were essentially the same thing and easily factored them into the same partial.